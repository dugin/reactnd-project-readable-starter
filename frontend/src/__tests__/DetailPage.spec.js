import React from 'react';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import DetailPage from "../components/detail-page/Detail.page";

describe('DetailPage Component', () => {

    const detailPage = shallow(<DetailPage/>);

    it('renders properly', () => {

        expect(toJson(detailPage)).toMatchSnapshot();

    });
});