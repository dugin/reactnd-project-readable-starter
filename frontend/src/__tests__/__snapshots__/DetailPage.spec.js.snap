// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailPage Component renders properly 1`] = `<div />`;
